.github/pull_request_template.md

```diff
@@ -7,14 +7,15 @@ Please include a summary of the changes and the related issue/task. Please also
 Please include the Jira Ticket Number and Link for this issue/task.
 
 ## Checklist before requesting a review
```

Comment:
why so many files changed?

Reply:
it just spacing based on prettiers output